mods/key_value.js

```diff
@@ -31,7 +31,7 @@ KVBucket.prototype.getBucketInfo = function(restbase, req, options) {
 };
 
 KVBucket.prototype.makeSchema = function(opts) {
-    opts.schemaVersion = 1;
+    opts.schemaVersion = 2;
```

Comment:
You also need to add `version: opts.schemaVersion` to the returned object, otherwise the version is not propagated into the *table* module.